openshift/template.yaml

```diff
@@ -82,6 +82,9 @@ parameters:
 - name: LOG_LEVEL
   value: "info"
   required: true
+- name: LOG_FORMAT
+  value: "json"
+  required: false
```

Comment:
If you set `LOG_FORMAT` to `json` as default in the code, you don't need to create a new env variable in the cloud.